style/taxonomy.css

```diff
@@ -52,3 +62,79 @@
     }
   }
 }
+
+.c-settingsMenu {
+  margin-left: var(--spacing--small);
+  position: relative;
+
+  &__iconButton {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 23px;
+    width: 23px;
+    border-radius: 50%;
+    border: 1px solid;
+    line-height: normal;
+    margin-right: 5px;
```

Comment:
Some of these small padding/margins can probably be replaced by `calc(var(--spacing--small) / 2)`. We should probably add a new var `--spacing--xsmall` based on that calculation...